### PR TITLE
Edit pages of the Operations pages

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request edit pages of the Operations pages

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/system/data-export.html
https://docs.magento.com/user-guide/stores/security-two-factor-authentication-manage.html#:~:text=Select%20the%20user%20and%20open,Reset%20and%20OK%20to%20confirm

### CODE REVIEW

File: src/system/data-export.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Export
...
+Special characters—such as the equal sign, greater and less than symbols, single and double quotes, backslash, pipe, and ampersand symbols—can cause problems during data transfer. To ensure that such special characters are correctly interpreted, they can be marked as an `escape sequence.` For example, if the data includes a string of text such as code=`str`, code=`str2`, choosing to enclose the text in double quotes ensures that the original double quotes are understood to be part of the data. When the system encounters a double set of double quotes, it understands that the outer set of double quotes is enclosing the actual data.
```
Comment (dobooth) [inline src/system/data-export.md:7]: I don't think this is a proper change. It is saying: If you need to the actual string to be `code="str"`, then wrap it in double quotes: `"code="str""`. Note that those backticks are markdown ticks to display the string as code.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by ZhovnirMaria** [comment]
Hi, @dobooth please check, thanks ☺️

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the branch with the code is successfully imported into the `magento-devdocs/merchdocs` repository. Branch name: `imported-ZhovnirMaria-merchdocs-1693`.

**Pull request merged**